commands/hset.md

```diff
@@ -11,4 +11,8 @@ If `field` already exists in the hash, it is overwritten.
 ```cli
```

Comment:
I don't understand the added value of this change to the example.

Reply:
`hset` can add multiple values. but the original explanation and example was showing only 1 field.
I wanted to demonstrate this command can add single and multiple fields without checking `hmset` command and its deprecation note.